test/testsuites/invalidopsrctests.go

```diff
@@ -78,9 +76,7 @@ func testOpSrcWithInvalidEndpoint(t *testing.T) {
 		}
 		return false, nil
 	})
-	if err != nil {
-		t.Errorf("OperatorSource never reached expected phase/message, expected %v: %v", expectedPhase, err)
-	}
+	assert.NoError(t, err)
```

Comment:
Why remove the message for this one? Does `assert.NoError` not support string formatting?

Reply:
@anik120 FYI my comment refers to each instance of this

Reply:
+1 on this. I thought @tkashem had already made this request in his review.